### PR TITLE
Add id

### PR DESCRIPTION
**Pull request by haneslinger**
#### Any background context you want to provide?

I forgot id the first time around

#### Screenshots (if appropriate)
image
image

### CODE REVIEW

File: seed/models/sensors.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class DataLogger(models.Model):
     )
 
     display_name = models.CharField(max_length=255)
+    identifier = models.CharField(max_length=2047, default="")
```
Comment (nllong) [inline seed/models/sensors.py:18]: For very long ids

**Review by nllong — APPROVED** [review]

**Comment by RDmitchell** [comment]
Can we call this field "Datalogger ID" ??

File: seed/models/sensors.py (hunk 1 above)
Comment (haneslinger) [reply seed/models/sensors.py:18]: 😬

**Comment by haneslinger** [comment]
> Can we call this field "Datalogger ID" ??

we can in the UI

**Comment by RDmitchell** [comment]
Datalogger ID in the UI is fine ... thx !

**Pull request merged**